existing-communities/README.md

```diff
@@ -1 +1,47 @@
-# Existing Communities
+# Communities
+A community is, according to the Oxford Dictionary,  *a group of people living in the same place or having a particular characteristic in common*{{"ComDef" | cite}}. In traditional research, the study focuses on localy bound communities like neighbourhoods and villages, because they have clear boundaries of who is involved in the community{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.  
+But thanks to modern media, like games, social media and messenger-applications, communities are not bound anymore by location and the characteristic becomes the central point of the community. A concequence of this is that boundaries of the community are less clear. Therefor, one of the challenges of doing research on these virtual communities is determining who is and who is not part of a community{{"sundaram2012understanding" | cite}}.  
+Virtual communities have four necessary characteristics: interactivity, communicators, a publicly shared mediated (virtual) communication place and a minimum level of sustained membership{{"jones1997virtual" | cite}}. This means that in a virtual community, there must be a longlasting, selfsustaining communication *(interactivity)* about subjects (related to the relating characteristics) by two or more persons *(communicators)* over one ore more platform like a website or a chat-group *(communication place)* where there is a certain amount of active members *(sustained membership)*.
+## Existing Communities
+Before we can compare different communities and their structure, we first need to define a few (types of) communities. We will look at two types of real-life communities and two types of virtual communities, such that we can discus their similarities and differences.
+### Ecosystems
+Ecosystems can be seen as a natural form of a community. It is bound to a location where a diversity of animals and species interact with eachother{{"EcosysDef" | cite}}. Although there is no real communication, the combination of individual self-interest (the survival of their offspring or themself) creates a network of dependencies and interactions that one can interpret as a community.  
+A key factor of an ecosystem is stability. In a stable ecosystem{{"EcosysSDef" | cite}}, each species acts in such a way that they, sometimes inderectly, enable or do not interfere with the survival of another species. An ecosystem will remain stable as long as the system is able to deal with outside disturbances.
```

Comment:
`they, sometimes inderectly, enable or do not interfere`

This is a weirdly constructed sentence in my opinion. They are indirectly enabling and/or indirectly not interfering? I would rephrase it to something like: `each species acts in such way that they don't interfere with the survival of another species, they sometimes even enable it unintentionally`